gp-includes/warnings.php

```diff
@@ -388,6 +473,140 @@ public function warning_should_not_end_on_newline( $original, $translation, $loc
 		return true;
 	}
 
+	/**
+	 * Adds a warning for changing plain-text URLs.
+	 * This allows for the scheme to change, and for some domains to change to a subdomain.
+	 *
+	 * @since 3.0.0
+	 * @access public
+	 *
+	 * @param string $original    The original string.
+	 * @param string $translation The translated string.
+	 * @return string|true        True if check is OK, otherwise warning message.
```

Comment:
```suggestion
	 * @return string|true True if check is OK, otherwise warning message.
```

`@return` and `@param` tags don't get aligned.

Reply:
Done in two methods.